app/helpers/recruitment_banner_helper.rb

```diff
@@ -0,0 +1,21 @@
+module RecruitmentBannerHelper
+  def there_are_banners?
+    return false if banners.nil?
+
+    true
+  end
+
+  def renderable_banners
+    current_url = request.original_url
+
+    banners.select do |banner|
```

Comment:
We won't ever show two banners on a page. So I think we should select the first banner that matches. Ie the order of the banners in the file is significant.